colossus-testkit/src/main/scala/colossus.testkit/ServiceSpec.scala

```diff
@@ -17,16 +17,17 @@ abstract class ServiceSpec[P <: Protocol](implicit clientFactory: FutureClientFa
   type Request  = P#Request
   type Response = P#Response
 
-  implicit val sys = IOSystem("test-system", Some(2), MetricSystem.deadSystem)
+  implicit val ioSystem: IOSystem = IOSystem("test-system", Some(2), MetricSystem.deadSystem)
 
   def service: ServerRef
-  def requestTimeout: FiniteDuration
+
+  def requestTimeout: FiniteDuration = 5.seconds
```

Comment:
Provided an implementation of this method so we don't need to implement in test cases. In general, people shouldn't care about it.